### PR TITLE
[Carry 17414] Added additional container information to "docker info".

### PR DESCRIPTION
**Pull request by calavera**
Carry #17414.

~~**WAIT BEFORE MERGE**: This PR is blocked by docker/engine-api#36~~

### CODE REVIEW

**Comment by calavera** [comment]
Vendoring is happening in #19243.

File: docs/reference/api/docker_remote_api.md [hunk 1]
```diff
@@ -112,6 +112,7 @@ This section lists each version from latest to oldest.  Each listing includes a
 * `GET /networks` now supports filtering by `name`, `id` and `type`.
 * `POST /containers/create` now allows you to set the static IPv4 and/or IPv6 address for the container.
 * `POST /networks/(id)/connect` now allows you to set the static IPv4 and/or IPv6 address for the container.
+* `GET /info` now includes the number of containers running, stopped and paused.
```
Comment (thaJeztah) [inline docs/reference/api/docker_remote_api.md]: should have a comma before `and paused.` (just a nit)

File: docs/reference/api/docker_remote_api_v1.22.md [hunk 2]
```diff
@@ -2081,6 +2081,9 @@ Display system-wide information
     {
         "Architecture": "x86_64",
         "Containers": 11,
+	"ContainersRunning": 7,
+	"ContainersStopped": 3,
+	"ContainersPaused": 1,
```
Comment (thaJeztah) [inline docs/reference/api/docker_remote_api_v1.22.md]: can you change these tabs to spaces?

**Comment by thaJeztah** [comment]
two really tiny nits, LGTM otherwise

**Comment by calavera** [comment]
fixed.

**Comment by thaJeztah** [comment]
LGTM

moving to "merge", taking previous LGTM's from https://github.com/docker/docker/pull/17414  into account

**Comment by thaJeztah** [comment]
:sob: janky, nooooo

`01:17:45 Build timed out (after 120 minutes). Marking the build as failed.`

**Pull request merged**